src/app/reducers/index.js

```diff
@@ -20,6 +20,7 @@ import meta from './meta';
 import messages from './messages';
 import modal from './modal';
 import moreCommentsRequests from './moreCommentsRequests';
+import optOuts from './optOuts';
```

Comment:
Was this checked-in from another commit? It looks like its missing in this patch